OptimizelySDKCore/OptimizelySDKCoreTests/OPTLYDecisionServiceTest.m

```diff
@@ -643,57 +643,84 @@ - (void)testGetVariationForFeatureWithTargetingRuleBucketing {
+    OCMVerify([bucketerMock bucketExperiment:fallBackRule withBucketingId:kUserId]);
     [bucketerMock stopMocking];
 }
 
-// should return variation when the user is bucketed into "Everyone Else" after attempting to bucket into all targeting rules
-- (void)testGetVariationForFeatureWithEveryoneElseRuleBucketingButNoTargetingRule {
+// should return variation when the user is bucketed into "Fall Back" after attempting to bucket into all targeting rules
+- (void)testGetVariationForFeatureWithFallBackRuleBucketingButNoTargetingRule {
     OPTLYFeatureFlag *booleanFeatureFlag = [self.config getFeatureFlagForKey:kFeatureFlagNoBucketedRuleRolloutKey];
     NSString *rolloutId = booleanFeatureFlag.rolloutId;
     OPTLYRollout *rollout = [self.config getRolloutForId:rolloutId];
-    OPTLYExperiment *everyoneElseRule = rollout.experiments[rollout.experiments.count - 1];
-    OPTLYVariation *expectedVariation = everyoneElseRule.variations[0];
-    OPTLYFeatureDecision *expectedDecision = [[OPTLYFeatureDecision alloc] initWithExperiment:everyoneElseRule
+    OPTLYExperiment *fallBackRule = rollout.experiments[rollout.experiments.count - 1];
+    OPTLYVariation *expectedVariation = fallBackRule.variations[0];
+    OPTLYFeatureDecision *expectedDecision = [[OPTLYFeatureDecision alloc] initWithExperiment:fallBackRule
                                                                                     variation:expectedVariation
                                                                                          source:DecisionSourceRollout];
     
     id bucketerMock = OCMPartialMock(self.bucketer);
-    OCMStub([bucketerMock bucketExperiment:everyoneElseRule withBucketingId:[OCMArg any]]).andReturn(expectedVariation);
+    OCMStub([bucketerMock bucketExperiment:fallBackRule withBucketingId:[OCMArg any]]).andReturn(expectedVariation);
     OPTLYDecisionService *decisionService = [[OPTLYDecisionService alloc] initWithProjectConfig:self.config bucketer:bucketerMock];
     
     // Provide null attributes so that user does not qualify for audience.
     OPTLYFeatureDecision *decision = [decisionService getVariationForFeature:booleanFeatureFlag userId:kUserId attributes:nil];
     
-    XCTAssertNotNil(decision, @"Get variation for feature with rollout having everyone else rule after failing all targeting rules should return variation: %@", decision);
+    XCTAssertNotNil(decision, @"Get variation for feature with rollout having fall back rule after failing all targeting rules should return variation: %@", decision);
     XCTAssertEqualObjects(decision.variation, expectedDecision.variation);
     
-    OCMVerify([bucketerMock bucketExperiment:everyoneElseRule withBucketingId:kUserId]);
+    OCMVerify([bucketerMock bucketExperiment:fallBackRule withBucketingId:kUserId]);
     [bucketerMock stopMocking];
 }
 
+- (void)testGetVariationForFeatureWithFallBackRuleBucketingId {
```

Comment:
I don't think this test is necessary as we shouldn't be respecting bucketingIds in rollouts